### ISSUE TITLE
Make Tedjin Confederacy use that title

### ISSUE THREAD

**Issue by Remerod** [MattTheLegoman/RealmsInExile#303]
So apparently paradox changed how ai view titular titles, they now prefer de jure a lot more, so like realms like the Tedjin Confederacy now prefers to form the de jure kingdom of Tedjin instead.

![image](https://user-images.githubusercontent.com/79544635/154940494-d04e7433-8de5-46f1-918f-3a8d6e4a292a.png)

**Comment by JukeJukestar** [comment]
Do note that this is a pretty stupid change and it isn't clear wether it was intended by PDX, so they might change it back in a patch sometime soon

**Comment by jj248** [comment]
This most certainly was intended by Paradox.  See the below from their patch notes for Royal Court on AI:

"The AI will now switch away from titular titles as its primary if it has a non-titular title of the same tier. It will also prefer its primary be a head of faith title if possible."

What's worse is that collective modding wisdom is that this is hardcoded:

![image](https://user-images.githubusercontent.com/26043074/155420279-015ecf06-e305-4f25-b613-d4c54a43b20e.png)

Therefore I'm pushing this to v3.0 when hopefully they de-hardcode it!

**Comment by JukeJukestar** [comment]
We talked about this in VC a bit, if they dont de-hardcode it in the next update (which they probably wont since they didnt change it after Fates of Iberia) I came up with this solution for the Tedjin:

Remove the Tedjin Confederacy title, instead give them the Kingdom of Tedja and set the color to the old Confederacy one.
Have the COA and Name change to "Tedjin Confederacy" if the holders culture has Tedjin heritage. It should be based on heritage and not language so once Bôr converts to Sôriyan, its still called the Tedjin Confederacy.
From the limited amount of testing I have done, this wont make the AI hyperfocused on revoking the dejure capital in Arka.
This would require changing all the events regarding the confederacy to use the k_tedja instead.
